ddp-workspace/projects/ddp-dsm-ui/src/app/WELCOME/auth/auth.component.ts

```diff
@@ -43,12 +44,13 @@ export class AuthComponent implements OnInit {
   popUpIsShown = true;
   authError: Observable<string | null>;
   pickList$: Observable<any>;
+  private source$: Observable<StorageEvent>;
```

Comment:
Tiny change: let's remove this unused field.

Reply:
And imports as well (fromEvent and SessionService)

Reply:
Surprised the linter didn't catch that!  They're cleaned up now.